viadot/sources/cloud_for_customers.py

```diff
@@ -26,23 +23,24 @@ def __init__(
     ):
         """
         Fetches data from Cloud for Customer.
-
         Args:
             report_url (str, optional): The url to the API in case of prepared report. Defaults to None.
             url (str, optional): The url to the API. Defaults to None.
             endpoint (str, optional): The endpoint of the API. Defaults to None.
             params (Dict[str, Any]): The query parameters like filter by creation date time. Defaults to json format.
             env (str, optional): The development environments. Defaults to 'QA'.
+            credentials (Dict[str, Any], optional): The credentials are provided from environmental variable
+            in local_config or with this parameter. Defaults to None than use credentials from local_config.
```

Comment:
Config doesn't hold or populate environment variables, should be: `credentials are populated with values from config file or this parameter`.